examples/prisma-example/apps/web/src/pages/profile/index.page.tsx

```diff
@@ -0,0 +1,123 @@
+import { NextPage } from 'next';
+import Head from 'next/head';
+import { Button, PasswordInput, Stack, TextInput, Title } from '@mantine/core';
+import { showNotification } from '@mantine/notifications';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { User } from "database";
```

Comment:
```suggestion
import { User } from 'types';
```

enable prettier & eslint and run `pnpm prettier`